script/apps/Aries/Creator/Game/Common/Files.lua

```diff
@@ -28,6 +28,9 @@ local EntityManager = commonlib.gettable("MyCompany.Aries.Game.EntityManager");
 
 local Files = commonlib.gettable("MyCompany.Aries.Game.Common.Files");
 
+--
+Files.prioritySearchDatas = {};
```

Comment:
Is there a way to clear the search data when another world is loaded or a different user signed in ?